iris/sentryiris.go

```diff
@@ -114,6 +114,10 @@ func GetHubFromContext(ctx iris.Context) *sentry.Hub {
 	return nil
 }
 
+func SetHubOnContext(ctx iris.Context, hub *sentry.Hub) {
+	ctx.Values().Set(valuesKey, hub)
```

Comment:
@cleptric The warning seems like a false negative, since I added there are tests for this.

Reply:
Which warning?

Reply:
![CleanShot 2024-12-17 at 10 56 23@2x](https://github.com/user-attachments/assets/7c3d5219-77e3-4b6e-ac57-a661ffb130ee)